0006/README.md

```diff
@@ -0,0 +1,319 @@
+-   Versions don't share any versioned file, nor URL, even if the content may be
+    the same. For example, `_site/v1.0/levels` and `_site/v2.0/levels` are
+    different.
+
+*What is changing*
+
+-   `https://slsa.dev/spec/v1.0/<page>` is renamed to
+    `https://slsa.dev/v1.0/<page>`.
+-   `https://slsa.dev/provenance/v1/<page>` is renamed to
+    `https://slsa.dev/v1.0/provenance`, but the predicateType remains the same.
+-   `https://slsa.dev/verification_summary/v1/<page>` is renamed to
+    `https://slsa.dev/v1.0/verification_summary`, but the predicateType remains
+    the same.
+-   Redirects exist from old to new pages, so URLs don't break.
+-   Using
+    [the proposed build process](#automate-building-and-deploying-the-website),
+    a version directory may contain files that are specific to the version, for
+    example `/images`.
+
+### The Repo
+
+#### Example
+
+```
+slsa-framework/slsa:main
+├─ docs/
+│  ├─ _data/
+│  ├─ _includes/
+│  ├─ _layout/
+│  ├─ _posts/
+│  ├─ _sass/
+│  ├─ fonts/
+│  ├─ images/  # contains shared images for the site.
+│  ├─ vendor/
+│  ├─ v0.1/    # a submodule points to the branch `release/v0.1`.
+│  ├─ v1.0/    # a submodule points to the branch `release/v1.0`.
+│  ├─ .ruby-version
+│  ├─ _config.yml
+│  ├─ Gemfile
+│  ├─ Gemfile.lock
+│  ├─ README.md
+│  └─ <other-unversioned-files>.md
+├─ .gitignore
+├─ .gitmodules  # uses `url = ./` to point to the local repo.
+├─ LICENSE.md
+├─ README.md
+└─ ...
+
+# Files were formerly in main/docs/*/v1, such as:
+# `docs/spec/v1.0/*`,
+# `docs/provenance/v1/*`,
+# `docs/verification_summary/v1/*`
+slsa-framework/slsa:release/v1.0
+├─ images/     # contains images specifically for this version.
+└─ <v1.0-versioned-files>.md
+
+# Files were formerly in main/docs/*/v0.1, such as:
+# `docs/spec/v0.1/*`,
+# `docs/provenance/v0.1/*`,
+# `docs/verification_summary/v0.1/*`
+slsa-framework/slsa:release/v0.1
+├─ images/     # contains images specifically for this version.
+└─ <v0.1-versioned-files>.md
+```
+
+#### Main Branch
+
+The directory structure of the `main` branch stays the same as the repo has
+right now.
+
+*What is changing*
+
+-   `/docs/spec`, `/docs/provenance`, `/docs/verification_summary`, and
+    `/docs/github-actions-workflow` are removed from the main branch.
+-   `/docs/images` doesn't contain versioned files and directories anymore. They
+    are removed or moved to their branch.
+-   `/docs/v1.0` and `/docs/v0.1` are two submodules that point to the
+    [version branches](#version-branches). As a result, `.gitmodules` is added
+    in the top level directory, and has `url = ./` for each submodule config to
+    ensure referencing to the local repo. A future version should follow a
+    similar setup in the repo.
+-   All permalinks in file content use the dir structure described in
+    [the website](#the-website) section.
+
+NOTE: We admit the way of using the Git submodule is non-standard, but it is a
```

Comment:
I am sorry, I cannot find any docs that looks authorized to me from Google or Internal Google. It may because I am new to Git/GitHub and the open source world. Would you have a recommendation here?

In general, I feel it is best not to use submodule to update code. I added another two new sub-sections Code Contribution and Submodule Pointer. I hope they could be helpful to clarify my thoughts on my mind right now.

Reply:
Updated the content to propose git subtree instead. I will resolve this comment.